docs/advanced-config/README.md

```diff
@@ -10,6 +10,78 @@ The configurations in this section are subject to change in future releases of S
 
 :::
 
+## TransientPrompt and TransientRightPrompt in Cmd
```

Comment:
I'm afraid single-shell functionality like this is strictly out of scope for Starship.

Reply:
As I said, it doesn't really have anything to do with Starship. Starship just provides a pretty formatted text to use, even if I add a functionality in Starship, called `--transient` (similar to `--right`, which at the moment is only supported by 3 out of 9 shells), it will be upto the shell to employ it.

Zsh already supports it (but someone needs to enable it in the init script), and it was added to Fish recently too - https://github.com/fish-shell/fish-shell/issues/7602 (not sure if it's released or not).

Besides, doesn't the Advanced Config section of Readme already have features that are present in only 1-2 shells? I can only see Bash, Zsh and PowerShell there.

Reply:
I'm of a mind with @matchai on this, at least as far as this file goes. Most other sections here are 20-40 line vignettes with a fair bit of whitespace, for functionality that is at least present in one or two other shells. This is 70 lines of fairly-dense text (dense = "not a lot of whitespace", not "hard to read"), which inflates the file line count by almost 50%, for details of how to do a single feature in a single shell.

I think this is important documentation to have, I just don't think this file is the right place for it in this PR. There are a few places we could put this:

  - Abuse the per-platform nature of the `Advanced Installation` guide to add this there (though I don't think that's a great idea either)
  - Separate it into a GitHub discussion thread
  - If the other shells are going to have similarly-complex configuration for transient mode, we should probably collect the documentation for all of them in a common place.

@matchai At the rate we're going, we're going to need some way to formalize the support differences between shells. Do you think now is a good time to set that up?

Reply:
@rashil2000 For the moment, is there a way you can preserve this text without introducing it into the PR? Heck, you can open a GitHub discussion/gist and just paste it there for now (and add a link here) if you're okay with that. I want this information to be preserved, I just don't think this file is the right place to be displaying it.

Reply:
I have reverted the commit